qiskit_quantuminspire/api/pagination.py

```diff
@@ -34,3 +34,11 @@ async def get_all(self, api_call: Callable[..., Awaitable[PageType]], **kwargs:
             if response.pages is None or page > response.pages:
                 break
         return items
+
+    async def get_single(self, api_call: Callable[..., Awaitable[PageType]], **kwargs: Any) -> Union[ItemType, None]:
+        """Get a single item from an API call that supports paging."""
+        response = cast(PageInterface[ItemType], await api_call(page=1, **kwargs))
```

Comment:
I think you don't have to specify page number here right? Shouldn't 1 be the default?

Reply:
You're right. No need to specify.